extensions/stackdriver/common/constants.h

```diff
@@ -13,6 +13,9 @@
  * limitations under the License.
  */
 
+#include <string>
+#include <string_view>
```

Comment:
FYI Envoy standardized on absl::string_view which is not aliased to std::string_view but easily convertible.

Reply:
I wouldn't mind replacing the `std::string_view` in this file